apps/docs/docs/reference/ui/atoms/Price.mdx

```diff
@@ -1,15 +1,30 @@
 ---
 description: Prices are used to display the price of a given product, discount and total values.
-sidebar_custom_props: 
+sidebar_custom_props:
   image: https://vtexhelp.vtexassets.com/assets/docs/src/Price___9acce93fd9399d7dcea4a3bb8b7de0c6.png
 ---
 
-import PropsSection from "@site/src/components/PropsSection/PropsSection";
+import PropsSection from '@site/src/components/PropsSection/PropsSection'
+import { TokenTable, TokenRow, TokenDivider } from '@site/src/components/Tokens'
 
 # Price
 
 Prices are used to display the price of a given product, discount and total values.
 
+## Overview
```

Comment:
I guess the sections order are a bit off. I would bring props and design tokens up.